incident_response_checklist.md

```diff
@@ -29,8 +29,7 @@ For on-call engineers
         - Enter how it was resolved
         - Gather logs
     - Set Origin: Automated alerting systems such as New Relic or human (Support team, CS)
-- <input type='checkbox'> **Create a post-mortem notes in the [Post Mortems Folder](https://drive.google.com/drive/u/0/folders/1-tGmN1KZqDIBePxc-ZleNv4meXoyA9ET) using the [template](https://docs.google.com/document/d/1BLtWVeWrzCpfEZSjSlSnTeUsJSB64rEd2WkSUgP7FZc/edit)**
-    - Use the date and a summary of the incident as the title
-    - Fill out a draft of the incident timeline, to the best of your ability
-    - Include a list of people who were heavily involved in remediation and should be present at the post-mortem
-    - Add a link to the notes in the post-mortem epic description 
+- <input type='checkbox'> **Create a [new post mortem](https://broadworkbench.atlassian.net/wiki/spaces/AP/pages/702021656/Post+Mortems) from the Confluence template.**
```

Comment:
would be nice to somehow indicate the link can be accessed from the prod board so that people don't have to go to checklist to get the link every time 🤔